sample/src/androidTest/java/com/schibsted/spain/barista/sample/RecyclerViewTest.java

```diff
@@ -26,4 +27,10 @@ public void checkClickRecyclerViewItem_byPosition_atThree() {
     clickRecyclerViewItem(R.id.recycler, 3);
     assertDisplayed("Raspberry");
   }
+
+  @Test
+  public void checkScrollToRecyclerViewItem_byPosition_atThree() {
+    scrollTo(R.id.recycler, 3);
```

Comment:
Yes! Add tons of elements, and then, do three tests: one to the first one, another test to an item of the center of the scroll, and another one to the last item of the list. In that way, we will be sure that it will always work :·)

By the way, thanks for the effort!